app/src/main/java/comcmput301f16t01/github/carrier/ElasticUserController.java

```diff
@@ -0,0 +1,99 @@
+package comcmput301f16t01.github.carrier;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import com.searchly.jestdroid.DroidClientConfig;
+import com.searchly.jestdroid.JestClientFactory;
+import com.searchly.jestdroid.JestDroidClient;
+
+import java.io.IOException;
+
+import io.searchbox.core.DocumentResult;
+import io.searchbox.core.Index;
+import io.searchbox.core.Search;
+import io.searchbox.core.SearchResult;
+
+/**
+ * Created by Ben on 2016-10-27.
+ */
+
+public class ElasticUserController {
```

Comment:
So I'm kind of just comparing this to the LonelyTwitter one because that's how much I know about elastic search.